azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +195,41 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
if there is only one possible value should be default it to "dst" and mark it optional?

or even hardcode this and remove it for now?

Reply:
I made it optional defaulting to "dst".